components/sponsored_benefits/Gemfile.lock

```diff
@@ -327,19 +325,19 @@ GEM
     zeitwerk (2.6.16)
 
 PLATFORMS
-  ruby
+  arm64-darwin-23
```

Comment:
did you do this chance consciously ? or it was done by bundler ? this will create some issues deploying